tests/cross/src/test/scala/tests/tokens/SemanticTokensSuite.scala

```diff
@@ -6,26 +6,28 @@ class SemanticTokensSuite extends BaseSemanticTokensSuite {
 
   check(
     "class, object, var, val(readonly), method, type, parameter, String(single-line)",
-    s"""|<<class>>/*keyword*/  <<Test>>/*class*/{
```

Comment:
Could you add a comment that the non symbol tokens are added outside the compiler?